app/helpers/ops_helper.rb

```diff
@@ -124,35 +124,46 @@ def server_zone_title_text
     _("The server zone cannot be changed when running in containers") if server_zones.length > 1 && !MiqServer.zone_is_modifiable?
   end
 
+  def ops_cell_onclick(node_key)
```

Comment:
Hard to tell in the diff, but this should be a private if it's not already.

Reply:
Will handle this in a different PR. There are a few methods to be moved to private.